terraform/environments/hmpps-oem/locals.tf

```diff
@@ -40,6 +39,58 @@ locals {
   baseline_ec2_autoscaling_groups        = {}
   baseline_ec2_instances                 = {}
   baseline_iam_policies = {
+    OracleEnterpriseManagementSecretsPolicy = module.baseline_presets.iam_policies_all.OracleEnterpriseManagementSecretsPolicy,
```

Comment:
If we set enable_ec2_oracle_enterprise_managed_server = true  then we don't need this.